Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -4042,7 +4042,7 @@ class RealtimeClientConnectionTests: XCTestCase {
         try testMovesToDisconnectedWithNetworkingError(NSError(domain: "NSPOSIXErrorDomain", code: 50, userInfo: [NSLocalizedDescriptionKey: "shouldn't matter"]), for: test)
     }
 
-    func test__102__Connection__Host_Fallback__should_move_to_disconnected_when_there_s_no_internet__with_any_kCFErrorDomainCFNetwork() throws {
+    func skipped_test__102__Connection__Host_Fallback__should_move_to_disconnected_when_there_s_no_internet__with_any_kCFErrorDomainCFNetwork() throws {
```

Comment:
I'm curious why this test was failing? Shouldn't the connection become DISCONNECTED per [RTN14d](https://sdk.ably.com/builds/ably/specification/main/features/#RTN14d)?